.github/workflows/cross-merge.yml

```diff
@@ -0,0 +1,79 @@
+name: Cross merge
+
+on:
+  workflow_call:
+    secrets:
+      robot-token:
+        description: 'A PAT token used by the app/robot'
+        required: true
+
+jobs:
+  job:
+    runs-on: ubuntu-latest
+
+    steps:
+        - uses: actions/checkout@v2
+          with:
+            fetch-depth: 0
+            token: ${{ secrets.robot-token }}
+
+        - name: Inject slug/short variables
+          uses: rlespinasse/github-slug-action@v3.x
+
+        - name: Get merge map
+          uses: octokit/request-action@v2.x
+          id: map
+          # Allow failures to be ignored.
+          # We will consider output as empty
+          continue-on-error: true
+          with:
+            owner: ibexa
+            repo: cross-merge-map
```

Comment:
Can we re-use the maintainers repository instead of introducing new one? I think that internal tools repository should be available to all Ibexa developers and common.

Reply:
Is this about using a separate repository or about the access to the new one?

Reply:
It's about this being in Yet Another Config Only Repository.

From my perspective, it's easier to remember to update one repository than multiple. Security-wise there is no reason to split them - those are company-wide information: 
 * who maintains which package
 * which repository was renamed into which
 * which package versions correspond to which Ibexa DXP package versions

Reply:
Agreed in private conversation to postpone this topic.